src/clusterfuzz/_internal/platforms/android/sanitizer.py

```diff
@@ -54,6 +54,12 @@ def get_options_file_path(sanitizer_tool_name):
   sanitizer_directory = ('/system' if settings.get_sanitizer_tool_name() else
                          constants.DEVICE_TMP_DIR)
 
+  # Reduce the log severity for HWASAN
+  # TODO(crbug.com/329346758) properly handle HWASAN options.
+  if sanitizer_tool_name.lower() == "hwasan":
+    logs.log('Unsupported sanitizer: ' + sanitizer_tool_name)
```

Comment:
Can you use an f-string?

Reply:
Can you change the log to be more indicative of the issue here?

We _do_ support hwasan as a sanitizer, but from what I can see, we _do not_ support setting additional `HWASAN_OPTIONS` via a metadata file.

So maybe:

```
    logs.log('Unable to set additional sanitizer options for HWASAN: ' + sanitizer_tool_name)
```

Reply:
+1. I'll modify the existing log and this diminished severity one.